js/main.js

```diff
@@ -26,4 +26,12 @@ $(document).ready(function () {
 			parent.removeClass('search-box--filled');
 		});
 	}
+
+	// Carousel Sidebar
+	const sidebarCarousel = $('#sidebarCarousel')
```

Comment:
create a class specific for this, remove the carousel class otherwise the js for the carousel runs also for desktop, even if you showed the other items with d-block (you can remove those classes as well). Since it's an exception only for mobile we initialise the carousel for the sidebar only on mobile. Actually the carousel on mobile is initialise even without this below code